pkg/trace/sampler/prioritysampler.go

```diff
@@ -58,7 +58,7 @@ func NewPrioritySampler(conf *config.AgentConfig, dynConf *DynamicConfig) *Prior
 	s := &PrioritySampler{
 		agentEnv:      conf.DefaultEnv,
 		localRates:    newSampler(conf.ExtraSampleRate, conf.TargetTPS, []string{"sampler:priority"}),
-		remoteRates:   newRemoteRates(conf.RemoteSamplingClient, conf.MaxRemoteTPS, conf.AgentVersion),
+		remoteRates:   nil,
```

Comment:
Any chance we can completely remove the concept of remote rates from the PrioritySampler? And rename local rates? I think it is generally better design to keep this sampler simple and control it from the outside (here your remote config listener/handler)